### ISSUE TITLE
Error analyzing MS-SQL database:  "The multi-part identifier "stats.object_id" could not be bound."

### ISSUE THREAD

**Issue by cwilso03** [janmotl/linkifier#6]
I'm getting the following error when trying to analyze a SQL Server database:

![Capture](https://user-images.githubusercontent.com/21103442/73071683-6a19cb00-3e81-11ea-920a-aafbd4f0055d.JPG)

I've tried updating statistics in the tables of that particular schema, but it made no difference.  Any thoughts on what to try next or how to resolve the issue?

**Comment by janmotl** [comment]
The issue is that Linkifier generated an SQL query that was not understood by the database.
1. Check that table `sys.stats` exists and that you can read from it.
2. Check that the table contains `object_id` column.
3. Check that the following query from [MSSQL.java](https://github.com/janmotl/linkifier/blob/master/src/vendor/MSSQL.java) works on the database:
```sql
SELECT TOP 100 
  TABLES.TABLE_NAME,
  stats.name 
FROM sys.stats
JOIN INFORMATION_SCHEMA.TABLES 
ON OBJECT_NAME(stats.object_id) = TABLES.TABLE_NAME 
```
4. If the snippet above works, test the whole query from `MSSQL.java`.
5. I am running out of ideas. Check whether there isn't `stats` table in the examined schema - the tables could have got somehow mixed... if that's the case, try to exclude "stats" in Linkifier's regex filter.

Let me know the outcome.

**Comment by cwilso03** [comment]
I was able to get the snippet query to work, but only after I modified it to give alias names to the `sys.stats` and `INFORMATION_SCHEMA.TABLES` tables, like so:

```sql
SELECT TOP 100 
  TABLES.TABLE_NAME,
  stats.name 
FROM sys.stats AS stats
JOIN INFORMATION_SCHEMA.TABLES AS TABLES
ON OBJECT_NAME(stats.object_id) = TABLES.TABLE_NAME 
```

**Comment by janmotl** [comment]
Thanks. I made a new release. Let me know if that fixed the issue (and whether there isn't any new issue).

**Comment by cwilso03** [comment]
That worked!  Thanks!
